clang/include/clang/StaticAnalyzer/Core/AnalyzerOptions.def

```diff
@@ -189,20 +189,23 @@ ANALYZER_OPTION(
     "crosscheck-with-z3-eqclass-timeout-threshold",
     "Set a timeout for bug report equivalence classes in milliseconds. "
     "If we exhaust this threshold, we will drop the bug report eqclass "
-    "instead of doing more Z3 queries. Set 0 for no timeout.", 700)
+    "instead of doing more Z3 queries. On fast machines, 700 is a sane value. "
+    "Set 0 for no timeout.", 0)
```

Comment:
```suggestion
    "instead of doing more Z3 queries. Setting this to 700 ms in conjunction with "
    "\"crosscheck-with-z3-timeout-threshold\" of 300 ms, would nicely guarantee "
    "that no bug report equivalence class can take longer than 1 second, "
    "effectively mitigating Z3 hangs during refutation. "
    "Set 0 for no timeout.", 0)
```